fluent-package/apt/systemd-test/test.sh

```diff
@@ -0,0 +1,52 @@
+#!/bin/bash
+
+if [ -z $1 ]; then
+    echo "Error: Need to specify $1 with VM name in the Vagrantfile."
+    echo "Ex.) $ ./test.sh ubuntu-focal"
+    exit 1
+fi
+
+vm=$1
+dir="/vagrant/fluent-package/apt/systemd-test"
+
+vagrant status $vm | grep -E "^${vm}\s+not created (.*)$"
+if [ $? -ne 0 ]; then
+    echo "Error: The VM already exists. Need to destroy it in advance with the following command."
+    echo "$ vagrant destroy $vm"
+    exit 1
+fi
+
+set -exu
+
+vagrant up $vm
+
+vagrant ssh $vm -- $dir/setup.sh
+vagrant ssh $vm -- $dir/install-newly.sh
+
+# I want to use snapshot instead of destorying it for every test,
+# but somehow, it will be often an error on GitHub Actions...
+#
+# $ vagrant ssh $vm -- $dir/setup.sh
+# $ vagrant snapshot save -f $vm after-setup
+# (execute a test)
+# $ vagrant snapshot restore $vm after-setup
+#
+# Error on GitHub Actions:
+# ==> debian-bullseye: Booting VM...
+# There was an error while executing `VBoxManage`, a CLI used by Vagrant
+# for controlling VirtualBox. The command and stderr is shown below.
+# Command: ["startvm", "4099f22d-a3b5-4c56-8d80-bf17b78bd8a8", "--type", "headless"]
+# Stderr: VBoxManage: error: ahci#0: The target VM is missing a device on port 0. Please make sure the source and target VMs have compatible storage configurations [ver=9 pass=final] (VERR_SSM_LOAD_CONFIG_MISMATCH)
+# VBoxManage: error: Details: code NS_ERROR_FAILURE (0x80004005), component ConsoleWrap, interface IConsole
+
+vagrant destroy -f $vm
+vagrant up $vm
+vagrant ssh $vm -- $dir/setup.sh
+vagrant ssh $vm -- $dir/update-from-v4.sh
+
+vagrant destroy -f $vm
+vagrant up $vm
+vagrant ssh $vm -- $dir/setup.sh
+vagrant ssh $vm -- $dir/update-to-next-version.sh
+
+vagrant destroy -f $vm
```

Comment:
How about simplifying with for loop?

```
for f in install-newly.sh update-from-v4.sh update-to-next-version.sh; do
  vagrant up $vm
  vagrant ssh $vm -- $dir/setup.sh
  vagrant ssh $vm -- $dir/$f
  vagrant destroy -f $vm
done
```

Reply:
Cool!! Thanks! I will fix it!

Reply:
Fixed! Thanks!

https://github.com/fluent/fluent-package-builder/compare/6a1b56ea4f26d59fdd2adb7e1ae24860cc8f5f62..3942824e9f70f0143683c9f37f9c2c937b07919b